src/pudl/metadata/resources/eia860.py

```diff
@@ -476,6 +476,42 @@
         "sources": ["eia860", "eia923"],
         "etl_group": "eia860",
     },
+    "denorm_emissions_control_equipment_eia860": {
```

Comment:
Is there a reason you've only added the `denorm` table and not the normalized one?

Reply:
I guess I wasn't sure what the protocol was for our table storage yet. But I could easily add it.

Reply:
I added this table to the harvesting process!